plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -324,7 +324,9 @@ export class KubernetesFanOutHandler {
           (item.errors !== undefined && item.errors.length >= 1) ||
           (item.resources !== undefined &&
             item.resources.length >= 1 &&
-            item.resources.some(fr => fr.resources.length >= 1)),
+            item.resources.some(
+              fr => fr.resources && fr.resources.length >= 1,
```

Comment:
```suggestion
              fr => fr.resources?.length >= 1,
```

Reply:
Thanks @sblausten, that one is checked already on line 325.

Reply:
Sorry. I understand now. Thanks for the feedback.